kubeconfig.go

```diff
@@ -36,18 +36,22 @@ func GetCertCaBase64(url string, client *http.Client) (ret string, err error) {
 	}
 
 	certs := resp.TLS.PeerCertificates
-	p, err := pemutil.Serialize(certs[0])
-	if err != nil {
-		return "", err
-	}
-	var buf bytes.Buffer
-	err = pem.Encode(&buf, p)
-	if err != nil {
+	if len(certs) > 1 {
+		p, err := pemutil.Serialize(certs[1])
```

Comment:
// Don't use certs[0], it is an ephemeral cert that gets rotated regularly.
		// Instead, certs[1] has a self-signed cert which we can hope remains relatively stable.